StackExchange.Redis/StackExchange/Redis/GeoUnit.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+
+namespace StackExchange.Redis
+{
+    /// <summary>
+    /// Units associated with Geo Commands
+    /// </summary>
+    public enum GeoUnit : short
+    {
+        /// <summary>
+        /// Meters
+        /// </summary>
+        m,
```

Comment:
Yes, it would need a corresponding tweak to the message generation; but the name of the enum here should **not** have to match what redis understands.